src/main/webapp/script.js

```diff
@@ -10,4 +10,28 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+function getYouTubeComments() { 
+  const urlInput = document.getElementById('url-entry');
+  var url = cleanseUrl(urlInput.value);
+  fetch("/YouTubeComments?url="+url)
+    .then(response => response.json()).then((comments) =>{
+      for(i = 0; i < comments.items.length; i++) {
+       //Todo: process comments
+      }
+    });
+}
+
+// Extracts video id from full url
+function cleanseUrl(url) {
+  // Split web address from parameters, extract first parameter
+  var videoId = url.split("?");
+  videoId = (videoId.length > 1) ? videoId[1].split("&")[0] : videoId[0];
+
+  // Remove parameter name to isolate video Id.
+  if(videoId.includes("v=")) { 
```

Comment:
Can simplify this a bit: .replace() does nothing if it can't find the right match, so you can get rid of the if statement.